app/config/config.yml

```diff
@@ -184,6 +184,7 @@ sonata_admin:
         groups:
             content:
                 label: Content
+                icon: '<i class="fa  fa-file-text-o"></i>'
```

Comment:
See http://stackoverflow.com/questions/11135261/should-i-use-i-tag-for-icons-instead-of-span for your first 2 questions.

The last question: Yes, it's the CSS CS I always follow. I'll remove them from this PR, to avoid confusion